content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -198,6 +205,11 @@ The port used to connect to DogStatsD, used in combination with `DD_AGENT_HOST`
 **Default**: `0`<br>
 Automatically flush the tracer when all the spans are closed; set to `1` in conjunction with `DD_TRACE_GENERATE_ROOT_SPAN=0` to trace [long-running processes][14].
 
+`DD_TRACE_APPEND_TRACE_IDS_TO_LOGS`
```

Comment:
Need to add `DD_LOGS_INJECTION` and maybe remove `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS` if we decide to

Reply:
Removed `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS` and its related doc (in its usage as well)